contents/docs/api/user.mdx

```diff
@@ -14,14 +14,14 @@ This includes all possible event names, event properties, API key, app URLs, cur
 
 
 ```shell
-GET /user
+GET /users/@me
 ```
 
 
 Example request:
 
 ```bash
-curl https://posthog.example.com/api/user/
+curl https://posthog.example.com/api/users/@me
```

Comment:
Nit but the trailing slash should stay in, since that's the way Django prefers things.